sorc/ncep_post.fd/MDL2SIGMA.f

```diff
@@ -277,8 +277,9 @@ SUBROUTINE MDL2SIGMA
            AHF  =D00
            FAC  =D00
 	   DONEFSL1=.TRUE.
-         ELSEIF(T(I,J,NL1XF(I,J))<SPVAL                 &
-     &	 .AND. Q(I,J,NL1XF(I,J))<SPVAL)THEN
+         ELSEIF(NL1XF(I,J)<LP1) THEN
+          IF(T(I,J,NL1XF(I,J))<SPVAL                 &
+     &	   .AND. Q(I,J,NL1XF(I,J))<SPVAL)THEN
```

Comment:
@SamuelTrahanNOAA Do you think two if statements can be combined?

Reply:
@SamuelTrahanNOAA Please add a change log with your name at the beginning session of this subroutine.

Reply:
No, they cannot be combined. Fortran has no short-circuit "and" operators. The `NL1XF(I,J)<LP1` must be checked before that index of an array is accessed. The only way to do that in Fortran is a nested "if" statement.

Reply:
I've added a comment to this effect.

Reply:
@SamuelTrahanNOAA I see it. Thanks!